graphlearn/examples/tf/ego_data.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2021 Alibaba Group Holding Limited. All Rights Reserved.
```

Comment:
ego_data.py ->ego_data_loader.py

Reply:
Done, renamed `ego_rgcn_data_loader.py` and `ego_sage_data_loader.py` as well.